mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/master/resourcecluster/ResourceClusterActor.java

```diff
@@ -454,6 +454,9 @@ private void onTaskExecutorAssignmentRequest(TaskExecutorAssignmentRequest reque
                 assignmentTimeout);
             sender().tell(matchedExecutor.get().getKey(), self());
         } else {
+            metrics.setCounter(
+                ResourceClusterActorMetrics.NO_RESOURCES_AVAILABLE,
+                TagList.create(ImmutableMap.of("resourceCluster", clusterID.getResourceID(), "workerId", request.getWorkerId().getId(), "jobCluster", request.getWorkerId().getJobCluster(), "jobId", request.getWorkerId().getJobId())));
```

Comment:
nit: do we need job cluster and job id both in there? Will job id alone be sufficient?

Reply:
good point. I think we can remove job cluster for now.